README.md

```diff
@@ -67,7 +67,7 @@ $ mvn test
 Run Individual Unit Test:
 
 ```bash
-cd related-module (e.g: pulsar-client)
+$ cd related-module (e.g: pulsar-client)
```

Comment:
nit: it would be great to put this change in different pull request. since it is an unrelated change.

Reply:
Sure, will be submitting separate PR for this.